docs/index.rst

```diff
@@ -1,32 +1,33 @@
+.. meta::
+  :description: TransferBench documentation
```

Comment:
Improved description. Use some of the content from opening paragraph

Reply:
Done.